notebooks/samples/explanations/ai-explanations-image-tf2.ipynb

```diff
@@ -0,0 +1,1758 @@
+{
```

Comment:
<p>Remove "Now let's look at the explanations"</p><p> </p><p>The images returned show the explanations for <strong>only the top class predicted by the model</strong>. This means that if one of the model's predictions is incorrect, the pixels you see highlighted are for the <em>incorrect class</em>. For example, if the model predicted "rose" when it should have predicted "tulip", you'll see explanations for why the model thought this image was a rose. </p><p> </p><p>If you deployed an <strong>Integrated Gradients model, </strong>you can visualize its feature attributions. Currently, the highlighted pixels returned from AI Explanations show the top 60% of pixels that contributed to the model's prediction. The pixels we'll see after running the cell below show us the pixels that signaled the model's prediction most.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>